examples/with-berry/packages/eslint-config/README.md

```diff
@@ -0,0 +1,3 @@
+# `@turbo/eslint-config`
```

Comment:
Those pesky README's!

```suggestion
# `@repo/eslint-config`
```

Reply:
Didn't notice auto merge was set, here's this one! https://github.com/vercel/turbo/pull/6607